### PR TITLE
Feature/exhibition series

### PR DESCRIPTION
**Pull request by CBU1980**
exhibitions

events

- Added initial list pages for events and exhibitions

### CODE REVIEW

**Comment by CBU1980** [comment]
ping @martzoukos

File: locales/de-DE.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@ de-DE:
     stories: gesichte
     partners: partners
   events:
+    list:
+      label: Zugehörige veranstaltungen
+      title: Neue veranstaltungen
+      image: /assets/images/exhibition-series.jpg
+      lead: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      paragraph_1:  Donec scelerisque bibendum est, sed semper dolor iaculis et. Nullam sit amet ex non libero malesuada ornare eget eget enim. Donec egestas nibh eu pharetra consectetur.
```
Comment (martzoukos) [inline locales/de-DE.yml]: User multi line text by starting the string to `|-`
```
      paragraph: |-
          Donec scelerisque bibendum est, sed semper dolor iaculis et. Nullam sit amet ex non libero malesuada ornare eget eget enim. Donec egestas nibh eu pharetra consectetur.
          Cras in feugiat elit, lobortis ornare magna. Curabitur consequat risus a leo sodales laoreet. Curabitur eu tincidunt risus.
```

And then in the view:
```
<%= partial 'partials/components/list-body', locals: {
     lead: t('events.list.lead'),
     text: t('events.list.paragraph')
   }
```

File: source/localizable/events/index.html.erb [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+<%= partial 'partials/components/list-header', locals: {
+    title: t('events.list.title'),
+    image: t('events.list.image'),
+    theme: 'red'
+  }
+ %>
+
+ <%= partial 'partials/components/list-body', locals: {
+     lead: t('events.list.lead'),
+     paragraph_1: t('events.list.paragraph_1'),
```
Comment (martzoukos) [inline source/localizable/events/index.html.erb]: Look at previous comment.

File: source/localizable/exhibitions/index.html.erb [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
+<%= partial 'partials/components/list-header', locals: {
+    title: t('exhibitions.list.title'),
+    image: t('exhibitions.list.image'),
+    theme: 'green'
+  }
+ %>
+
+ <%= partial 'partials/components/list-body', locals: {
+     lead: t('exhibitions.list.lead'),
+     paragraph_1: t('exhibitions.list.paragraph_1'),
```
Comment (martzoukos) [inline source/localizable/exhibitions/index.html.erb]: Similar.

**Pull request merged**